api/v1alpha2/cloudprovider_encoding_test.go

```diff
@@ -244,24 +244,24 @@ datacenter = us-west
 folder = kubernetes
 
 `,
-			configObj: cloudprovider.Config{
-				Global: cloudprovider.GlobalConfig{
+			configObj: v1alpha2.CloudProviderConfig{
+				Global: v1alpha2.CloudProviderGlobalConfig{
 					Datacenters:     "us-west",
 					SecretName:      "vccreds",
 					SecretNamespace: "kube-system",
 					CAFile:          "/some/path/to/my/trusted/ca.pem",
 				},
-				VCenter: map[string]cloudprovider.VCenterConfig{
+				VCenter: map[string]v1alpha2.CloudProviderVCenterConfig{
 					"0.0.0.0": {},
 					"1.1.1.1": {},
 				},
-				Workspace: cloudprovider.WorkspaceConfig{
+				Workspace: v1alpha2.CloudProviderWorkspaceConfig{
 					Server:     "0.0.0.0",
 					Datacenter: "us-west",
 					Folder:     "kubernetes",
 				},
-				ProviderConfig: cloudprovider.ProviderConfig{
-					Cloud: &cloudprovider.CloudConfig{
+				ProviderConfig: v1alpha2.CloudProviderProviderConfig{
```

Comment:
_Maybe_ CloudProviderConfigSpec?

Reply:
so on the CCM we use `CPIConfig` IIRC

@andrewsykim - we might want to standardize naming accross CPIs and other consumers.

Reply:
CPI sounds good to me as well, wdyt @noamran ?

Reply:
Sure! So just to be clear- we want to change all the prefixes to `CPI`  instead of `CloudProvider`? Or just for this type?

Reply:
use `CPI` as prefix